### ISSUE TITLE
Fails to calculate the correct gradient for oneflow.new_ones

### ISSUE THREAD

**Issue by xxxyyyzzz12345** [Oneflow-Inc/oneflow#8945]
## Summary
The following code snippet will produce "Check failed: outputs.at(i)->requires_grad() All output tensors `.requires_grad` should be true" error. However, the theoretical gradient should be 1. Furthermore, I set "requires_grad=True" and the "x.grad" will always be "none".
```
x = oneflow.tensor(np.ones((1, 1, 5))).requires_grad_()
y = oneflow.new_ones(x,(2, 2)) #requires_grad=True
oneflow.sum(y).backward()
print(x.grad)
```
